helloworld-jms/src/test/resources/helloworld-jms-logging.properties

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
do we really need logging properties on a helloworld? This quickstart should be as simple as possible, focusing on JMS only)